examples/ReactExample/components/todo-app.js

```diff
@@ -18,10 +18,13 @@
 
 'use strict';
 
-import React, {
+import React from 'react';
+
+import {
```

Comment:
What I mean is do we need to change this so that each import is separate? Isn't that what was deprecated with the latest release?